### ISSUE TITLE
"Save" icon wasn't selected on Mac

### ISSUE THREAD

**Issue by Helen09** [Microsoft/pxt#2967]
**Repro steps:**
• Navigate to https://makecode.microbit.org/beta on Firefox&Safari on Mac
• Click on "save" button

**Expected result:**
When click "save" button, "save" button should be selected with blue box
![sep-19-2017 16-59-53](https://user-images.githubusercontent.com/23466737/30584612-de9f27a4-9d5c-11e7-8b5e-98d55a447a65.gif)

**Actual result:(Firefox&Safari)**
When click "save" button, "save" button wasn't selected with blue box
![sep-19-2017 17-01-26](https://user-images.githubusercontent.com/23466737/30584660-fc12ba12-9d5c-11e7-9aa3-b81c91f41609.gif)

**Comment by microsoftsam** [comment]
This is due to the default Accessibility settings on mac. The issue you see here is that both Safari and Firefox on mac respect the system preferences for tab focus, and only focus on an element when it is tabbed to using a keyboard. Clicking on a button doesn't enable it's focus, which it does in other browsers. 

There's no real way around it, to read up more about the issue, here's a good explanation: 
https://stackoverflow.com/questions/11704828/how-to-allow-keyboard-focus-of-links-in-firefox/11713537#11713537
